ServerCore/Pages/Events/Standings.cshtml

```diff
@@ -43,7 +43,8 @@
             <tbody>
         @for (int y = 0; y < Model.Teams.Count; y++) {
             var team = Model.Teams[y];
-                <tr>
+            var finished = team.FinalMetaSolveTime < DateTime.MaxValue;
```

Comment:
Does this need to check on cheat code usage?

Reply:
FinalMetaSolveTime calculation already does.

                    FinalMetaSolveTime = g.Where(s => s.Puzzle.IsCheatCode).Any() ?
                        DateTime.MaxValue :
                        (g.Where(s => s.Puzzle.IsFinalPuzzle).Select(s => s.SolvedTime).FirstOrDefault() ?? DateTime.MaxValue)